packages/arb-ts/src/lib/bridge.ts

```diff
@@ -78,4 +97,11 @@ export class Bridge extends L2Bridge {
       destinationAddress
     )
   }
+  public getAndUpdateL1TokenData(erc20l1Address: string) {
+    return this.l1Bridge.getAndUpdateL1TokenData(erc20l1Address)
+  }
+
+  public async getAndUpdateL1EthBalance() {
```

Comment:
I believe this doesn't need to be async

Reply:
we want it async because we want to both save it to the state of bridge instance and return it; you're right that's un-idiomatic tho, will clean